### DISCUSSION TITLE
Frequently Asked Questions (FAQ) wanted

### DISCUSSION THREAD

**Discussion by JoostvanBruggen** [Mattk70/Chirpity-Electron#93] [Ideas]
I can imagine that a lot of the same questions are asked or obscured by a fague title.. It woulde be nice to prevent a lot of the same questions by making, or expanding a FAQ list. Maybe type in a word to find the likely question..

**Comment by Mattk70** [comment]
I wish they were Joost! I would much prefer too many questions than the very little detailed feedback I receive. I have been thinking that I could add a forum to the website to encourage more feedback and discussion. I could then link to the website forum from the application itself. It seems not so many people want to post here on github.

**Reply by JoostvanBruggen** [reply]
The "problem" with Github is that you'll need yet another account just to ask you a question and it does have a véry technical feel. I don't belong in such a code-writing world.. could be a first impression. A forum would be better I think.. a bit like they have on sites like Xeno-Canto..?
